src/main/java/com/faforever/client/player/PlayerService.java

```diff
@@ -97,14 +100,16 @@ public void onGameRemoved(GameRemovedEvent event) {
     Game game = event.getGame();
     ObservableMap<String, List<String>> teams = game.getTeams();
     synchronized (teams) {
-      teams.forEach((team, players) -> updateGamePlayers(players, null));
+      List<String> playersInGame = teams.entrySet().stream().flatMap(stringListEntry -> stringListEntry.getValue().stream()).collect(Collectors.toList());
```

Comment:
To be honest this is equivalent to the way it was done before but it is more convenient to do it in one call also for debug purposes